paper-form-fields.md

```diff
@@ -87,6 +91,55 @@ press the enter key. There's no default for this function.
 **escape()** You can pass a function called escape, that will be executed everytime
 you press the escape key. There's no default for this function.
 
-**format(value)** Gets the value and formats it, before updating the object.
-There's no default this function.
+**format(value)** Gets the value and formats it (only for displaying). There's no
+default function.
+
+**serialize(value)** Get the value and formats it before the actual value is
+updated. There's no default function.
+
+### check-box
+It renders a md-checkbox, which is actually _not_ a HTML input[type="checkbox"] element.
+This checkbox kan toggle a boolean and does not need any specific attributes. There're
+no attributes or methods to overwrite.
+
+### radio-group
+It renders a md-radio-group, including a md-radio-button for each given option. It does
+_not_ render an actual HTML input[type="radio"] element. There are no attributes or
+methods to override. Possible attributes and methods for this field include:
+
+**options (required)** This array can contain number, strings or objects. If you have an array
+objects, you need te pass a labelPath too.
+
+**labelPath** If the options contain objects, you have to pass a propertyPath as string
```

Comment:
In the component itself, you have a `labelPropertyPath` and `valuePropertyPath`. Keep the readme up-to-date with the component.